Shokofin/Resolvers/VirtualFileSystemService.cs

```diff
@@ -51,6 +51,10 @@ public class VirtualFileSystemService {
     // Note: Out of the 14k entries in my test shoko database, then only **348** entries have a title longer than 64 characters.
     private const int NameCutOff = 64;
 
+    private static readonly HashSet<string> AudioFileExtensions = [
```

Comment:
We don't want to keep our own list of file extensions when Jellyfin has it's own list of what it supports we can use. Said list is accessible as `NamingOptions.AudioFileExtensions` in this file.